airflow/www/extensions/init_wsgi_middlewares.py

```diff
@@ -37,8 +38,11 @@ def _root_app(env: WSGIEnvironment, resp: StartResponse) -> Iterable[bytes]:
 
 def init_wsgi_middleware(flask_app: Flask) -> None:
     """Handle X-Forwarded-* headers and base_url support."""
+    webserver_base_url = conf.get("webserver", "BASE_URL")
```

Comment:
```suggestion
    webserver_base_url = conf.get_mandatory_value("webserver", "BASE_URL", fallback="")
```

Otherwise the `endswith` check below would blow up.